docs/docs/single-tenant/fastapi_configuration.mdx

```diff
@@ -299,9 +347,9 @@ if settings.BACKEND_CORS_ORIGINS:
 azure_scheme = SingleTenantAzureAuthorizationCodeBearer(
     app_client_id=settings.APP_CLIENT_ID,
     tenant_id=settings.TENANT_ID,
-    scopes={
-        f'api://{settings.APP_CLIENT_ID}/user_impersonation': 'user_impersonation',
-    }
+    scopes=settings.SCOPES,
+    openapi_authorization_url=settings.OPENAPI_AUTHORIZATION_URL,
```

Comment:
These are new params added to [SingleTenant](https://intility.github.io/fastapi-azure-auth/single-tenant/fastapi_configuration) not previously in the docs. I don't have much experience with Single Tenant and would refer to @JonasKs